src/Evolution/Executables/ScalarWave/EvolveScalarWave.hpp

```diff
@@ -35,37 +35,40 @@
 
 template <size_t Dim>
 struct EvolutionMetavars {
+ private:
+  using temporal_id_tag = Tags::TimeId;
+
+ public:
   // Customization/"input options" to simulation
   using system = ScalarWave::System<Dim>;
-  using temporal_id = Tags::TimeId;
   using analytic_solution_tag =
       CacheTags::AnalyticSolution<ScalarWave::Solutions::PlaneWave<Dim>>;
-  using normal_dot_numerical_flux =
+  using numerical_flux_tag =
```

Comment:
Why are you changing the name of this? Is it important? The old name actually describes what it is, the new one is incorrect.